snapshot/lib/assets/SnapshotHelper.swift

```diff
@@ -7,7 +7,7 @@
 //
 
 // -----------------------------------------------------
-// IMPORTANT: When modifying this file, make sure to 
```

Comment:
The version is bumped up in this file, and there's a new `SnapshotHelperXcode8.swift`. I think when you run _snapshot_ for after updating _fastlane_ it will point you to the next `SnapshotHelperXcode8.swift`... I think, we need to test it 🍵